client/src/test/java/com/sinch/sdk/models/ConfigurationTest.java

```diff
@@ -22,6 +25,8 @@ class ConfigurationTest {
           .setNumbersUrl(NUMBERS_SERVER)
           .setSmsRegion(SMS_REGION)
           .setSmsUrl(SMS_SERVER)
+          .setApplicationKey(APPLICATION_KEY)
+          .setApplicationSecret(APPLICATION_SECRET)
           .build();
 
   @Test
```

Comment:
I took me a while to remember that the credential values should not appear as a result of the object conversion to a string. Do you think it's worth to rename the method such as `testToStringShouldNotDiscloseCredentialValues()` ?

Reply:
Don't the assert error messages was not enough from `testToString` to remind this ?

Reply:
With the method name, I was expecting that you would test the `toString()` against the expected result. This is why I was puzzled that you were doing some assertFalse. That's up to you ;)

Reply:
Yes; I understand your point of view.... and this is why the point of view to test what toString is doing is having sense... just to ensure it do not output too much information
To be honest; checking the output of toString  for another context could seems to be over-engineering